client/app/admin/admin.component.ts

```diff
@@ -0,0 +1,154 @@
+            role: 'Manager'
+        },
+        {
+            firstname: 'John',
+            surname: 'McGregor',
+            role: 'Manager'
+        },
+        {
+            firstname: 'Lucy',
+            surname: 'McCabe',
+            role: 'Chef'
+        }
+    ];
+
+    private displayOrdersTodayBool;
+    private displayOrdersWeekBool;
+    private displayOrdersMonthBool;
+
+    constructor(
+        protected adminService: AdminService,
+        private menuService: MenuService
+    ) { }
+
+    ngOnInit() {
+        this.getMenu();
+    }
+
+    getMenu() {
+        this.menuService.getMenu().subscribe(result => {
+            this.menu = result;
+        })
+    }
+
+    updateMenu(itemId, menuItem) {
+        //this.menuService.update(itemId, menuItem);
+    }
+
+    addItem() {
+        var itemName = this.removeBrackets(document.getElementById("itemName").innerHTML);
+        var itemPrice = this.removeBrackets(document.getElementById("itemPrice").innerHTML);
+        var itemCategory = document.getElementById("menuCategorySelect").value;
+        console.log(itemName);
+        console.log(itemPrice);
+        console.log(itemCategory);
+        if (this.isNumeric(itemPrice)) {
+            var newItem = { name: itemName, price: itemPrice, category: itemCategory };
+            this.menuService.create(newItem);
+            this.getMenu();
+        } else {
+            alert("Item price is not a valid number. Try again and add to menu.");
+        }
+    }
+
+    removeBrackets(string) {
+        var replaced = string.replace(/\<[^\>]*\>/, '');
+        return replaced;
+    }
+
+    isNumeric(num) {
+        console.log(!isNaN(num));
+        return !isNaN(num);
+    }
+
+    openMenuNav() {
+        document.getElementById("menuBar").style.width = "30%";
+        document.getElementById("menuBar").style.display = "block";
+        document.getElementById("menuMain").style.visibility = "hidden";
+        document.getElementById("summary").style.marginLeft = "20%";
+    }
+
+    closeMenuNav() {
+        document.getElementById("menuBar").style.width = "0";
+        document.getElementById("menuBar").style.display = "none";
+        document.getElementById("menuMain").style.visibility = "visible";
+        document.getElementById("summary").style.marginLeft = "0";
+    }
+
+    openStaffNav() {
+        document.getElementById("staffBar").style.width = "30%";
+        document.getElementById("staffBar").style.display = "block";
+        document.getElementById("staffMain").style.visibility = "hidden";
+        document.getElementById("analytics").style.marginLeft = "20%";
+    }
+
+    closeStaffNav() {
+        document.getElementById("staffBar").style.width = "0";
+        document.getElementById("staffBar").style.display = "none";
+        document.getElementById("staffMain").style.visibility = "visible";
+        document.getElementById("analytics").style.marginLeft = "0";
+    }
+
+    displayOrdersToday() {
```

Comment:
Not sure i get how this function works... 
isn't the code inside the if statement doing the same thing as the one from the else?

Reply:
Sorry, my eyes are just letting me down... is setting them the other way around.

Reply:
The 3 functions are doing the same thing, just for other ids, in that case it would be easier to just have 1 function, that takes in the ids and a bool. Again, there we could just use ngClass to change the css based on conditions, but we could add it later on i guess.